src/main/java/lovespiritual.java

```diff
@@ -0,0 +1,111 @@
+import java.util.Scanner;
+
+public class lovespiritual {
+    public static void main(String[] args) {
+        String line = "__________________________________________________"; // horizontal line
+        Scanner in = new Scanner(System.in);
+        String[] tasks = new String[100]; // array of tasks
+        boolean[] isMarked = new boolean[100]; // check if task is marked
+        String[] taskTypes = new String[100];
+        int taskCount = 0; // count the number of tasks added in the array
+
+        // introduction
+        System.out.println(line);
+        System.out.println("Hello! I'm lovespiritual");
+        System.out.println("What can I do for you?");
+        System.out.println(line);
+
+        // loop that keeps recurring when the program is running
+        while (true) {
+            String input = in.nextLine().trim();
+
+            if (input.equalsIgnoreCase("bye")) {
```

Comment:
This is hard for the reviewer to review due the nested if else statement (Arrow Head). Adding a switch case and separating each conditions as Methods.